woocommerce/woocommerce-ios
# [IPPInAppFeedback] Change merchant eligibility condition

Closes: #8680 


## Description
This PR changes one of the conditions for choosing which banner we will be showing to users based on their WCPay transactions. The new condition is:

- 2: Merchants who have completed their first transaction in WCPay, but less than 10 IPP transactions in the last 30 days in WCPay.

All groups must have COD enabled, and be on an IPP supported country.

## Changes:
- Renamed variables and methods for better description
- Modified the second condition to check if has more than one WCPay transaction, and if has less than 10 transactions if the last 30 days.

## Testing instructions:
1 - On a Store with COD enabled, and less than 10 IPP transactions:
2 - Add a breakpoint on OrderListViewModel's [line 333](https://github.com/woocommerce/woocommerce-ios/blob/1998b4eec9fbc40b63433a830b2222f080e8dbaa/WooCommerce/Classes/ViewRelated/Orders/OrderListViewModel.swift#L333) to intercept the case for "at least 1 transaction historically, but less than 10 transactions in the last 30 days"
3- Navigate to Menu > Orders
4- The breakpoint should be triggered
